Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Processes/Validation/DownWithOnOff.mo

```diff
@@ -138,6 +132,14 @@ protected
   Buildings.Controls.OBC.CDL.Logical.Latch lat
     "True when it is not in process"
     annotation (Placement(transformation(extent={{120,210},{140,230}})));
+  Buildings.Controls.OBC.CDL.Logical.Pre chiOneHea(
+    final pre_u_start=false)
+    "Chiller one head pressure control"
+    annotation (Placement(transformation(extent={{100,-100},{120,-80}})));
+  Buildings.Controls.OBC.CDL.Logical.Pre chiTwoHea(
+    final pre_u_start=true)
```

Comment:
Just out of curiosity, why is this being initialized to `true`?

Reply:
This validation was set to staging down the initial chiller stages `{true, false}`, to `{false, true}`, where the first chiller is larger than the second one. So, initially, the head pressure control of chiller one is enabled.

Reply:
...two. Got it! Thanks.